upgrade.sh

```diff
@@ -23,15 +23,15 @@ function waitForInstallPlan() {
 
 function waitForACMRegistryPod() {
     for i in `seq 1 30`; do
-    	oc get po -n $TARGET_NAMESPACE -lapp=acm-custom-registry -oyaml
-        oc get po -n $TARGET_NAMESPACE -lapp=acm-custom-registry -oyaml | grep "$NEXT_SNAPSHOT"
+    	oc get po -n openshift-marketplace -lolm.catalogSource=acm-custom-registry -oyaml
```

Comment:
Is there a reason why we didn't just update the TARGET_NAMESPACE variable or create a new one for this part?

Reply:
TARGET_NAMESPACE should stay the same. The subscription/CSV/installplan will all be stood up in the TARGET_NAMESPACE, whereas only the catalogsource is stood up in `openshift-marketplace`

Reply:
Do we want to make that another variable or leave it explicitly defined?